Various/BuyOne_Dynamic ReaperMenu (guide inside).lua

```diff
@@ -0,0 +1,254 @@
+--[[
+ReaScript name: Dynamic ReaperMenu
+Author: BuyOne
+Website: https://forum.cockos.com/member.php?u=134058
+Version: 1.0
+Changelog: Initial release
+Provides: [main=main,midi_editor] .
+About:
+
+#### GUIDE
+
+- The script is designed to allow using REAPER menus independently of REAPER
+menu system by loading .ReaperMenu files on demand, which essentially allows
+potentially having and using many more menus and toolbars than the REAPER
+native menu system provides.
```

Comment:
The content of this `About` tag is empty, because the following text is not indented.

Reply:
Thank you, by how much should it be indented?

Reply:
The amount of spaces or tabs doesn't matter, as long as it is consistent across all lines. I'd recommend using the package editor instead of manually writing the metadata and pull requests...

Reply:
OK, i think i'm more comfortable with this interface.